app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ContactsController < ApplicationController
+  
+  def new
+    @contact = Contact.new
+  end
+
+  def create
+    @contact = Contact.new(contact_params)
+    
+    if @contact.save
+      ContactsMailer.general_message(@contact).deliver
```

Comment:
is this working?
`#deliver` was deprecated some time ago.. is it still available?